### DISCUSSION TITLE
Deletes half a conversation

### DISCUSSION THREAD

**Discussion by Mirevien** [victornpb/undiscord#475] [4. Bugs and Issues]
[End condition B] if you see this please report.	{"running":true,"delCount":0,"failCount":0,"grandTotal":0,"offset":0,"iterations":1,"_seachResponse":{"total_results":0,"messages":[],"analytics_id":"cd9b4b8ff7921af1fe8c820665412003"},"_messagesToDelete":[],"_skippedMessages":[]}

Here, I dont know what to do

**Comment by victornpb** [comment]
#471
